### PR TITLE
postgis: fix build against postgres 13.0 and license

### PR DESCRIPTION
**Pull request by ruurd**
- [X ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [X ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
Retry this. Last PR was turning into mush so redid the changes branched from a more recent master

### CODE REVIEW

File: Formula/postgis.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@ class Postgis < Formula
   homepage "https://postgis.net/"
   url "https://download.osgeo.org/postgis/source/postgis-3.0.2.tar.gz"
   sha256 "a3a1641dfd73c83924088a185bdb8b35567b3d1dc8d0887f9e4b492e228ab2ca"
+  license "GPL-2.0-or-later"
+  revision 1
```
Comment (chenrui333) [inline Formula/postgis.rb:7]: can you amend your commit message as `postgis: <why need a revision bump>`?

Comment (ruurd) [reply Formula/postgis.rb:7]: Did not build against postgresql 13.0, so bumped rev on request.
Added license conforming to what is mentioned on the postgis homepage.

**Comment by ruurd** [comment]
OK. Same problem as earlier...

  with:
    token: ***
    failure_label: automerge-skip
Error: Cannot read property 'filename' of undefined

**Comment by ashkulz** [comment]
@ruurd: you may need to rebase/squash your commit with message `postgis: bump revision to rebuild with postgresql 13`, as [this line in check-commit-format](https://github.com/Homebrew/actions/blob/master/check-commit-format/main.js#L46) seems to be the problem.

**Comment by fxcoudert** [comment]
I have rebased/squashed this, so we can merged ASAP, since @ruurd seemed to have trouble with the rebasing

**Review by fxcoudert — APPROVED** [review]

**Comment by ruurd** [comment]
Thank you @fxcoudert sometimes I have a huge fight with git for some reason or other 😢

**Comment by ruurd** [comment]
@fxcoudert seems to me something is hanging again here... CI job for 10.15 won't start

File: Formula/postgis.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/postgis.rb:7]: Sounds good  👍

**Review by chenrui333 — APPROVED** [review]

**Comment by ruurd** [comment]
OK.... sometimes it ain't easy. Thank you everyone for helping this come to an end.

**Comment by chenrui333** [comment]
@ruurd thanks for your efforts on this! Appreciate it!

**Comment by ruurd** [comment]
Thank you. I have an interest in PostgreSQL so assisted in fixing that then this happened as collateral effort. And since I am an avid user of Homebrew I think I am obliged to contribute if I can.

**Pull request closed without merging**